### PR TITLE
Migrate to gradle build system.

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:**
https://jira.appcelerator.org/browse/MOD-2563

**Note:**
I tried to get update to using AndroidX in the source files, but unfortunately it is not only a namespace change. I think it will bloat unnecessary this task and it will be for the best if we separate an update in another one.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -0,0 +1,5 @@
+dependencies {
+	implementation 'com.google.zxing:core:3.4.0'
+	implementation 'com.google.zxing:android-core:3.3.0'
+	implementation 'com.google.zxing:android-integration:3.3.0'
+}
```
Comment (jquick-axway) [inline android/build.gradle]: Maybe we shouldn't bother with a "build.gradle" for this module.

The zxing library does not offer an AAR release. It only provides JARs for the 3 libraries you've listed above... and we have a copy of the "source" tree and "res" tree of the main zxing "android" library below which provides the activities and other things... which the zxing does not provide a "release" for.
https://github.com/appcelerator-modules/ti.barcode/tree/master/android/src/com/google/zxing/client/android
https://github.com/appcelerator-modules/ti.barcode/tree/master/android/platform/android/res

This means if we want to update the zxing JAR versions, then we also have to update replace the above mentioned source trees like how it was done in "ti.barcode" here...
https://github.com/ypbnv/ti.barcode/commit/03c7cb19234d653f2c35762e902922d0fbbd3353#diff-3aaa7668bb33c61a18f41519e5497018

Yordan, the zxing libraries you've listed above are older than what's currently committed. How about we delete the "build.gradle" file and keep the "lib" JARs. That is, don't update the zxing library.

Comment (ypbnv) [reply android/build.gradle]: I agree, I did not think about updating in the future. Updated.

**Comment by sgtcoolguy** [comment]
I think #112 should be considered before this one, seeing as it may impact things.

**Comment by jquick-axway** [comment]
@sgtcoolguy, agreed. How about we go ahead and merge PR #112 and then test Android/iOS based on this PRs final changes?

**Pull request merged**